### ISSUE TITLE
Support config file?

### ISSUE THREAD

**Issue by mritd** [riobard/go-shadowsocks2#26]
Can we add profile support?

Using the Cobra library to load configuration files and process command line arguments seems more friendly; if possible, I will commit a PR.

**Comment by riobard** [comment]
Do you mean configuration file?

**Comment by mritd** [comment]
right

**Comment by riobard** [comment]
Yeah… that's the tricky part. Many people have requested support for config files. I'm hesitated to do so because the feature set is not stable yet. Additionally, there are differences to other SS implementations that reusing config file format is questionable. I'll keep this issue open until a better understanding is reached.
